packages/klevu-ui/src/components/klevu-chat-messages/klevu-chat-messages.tsx

```diff
@@ -78,6 +81,9 @@ export class KlevuChatMessages {
             const givenFeedback = this.feedbacks?.find((f) => f.id === message.message.id)
             const isLastMessage = this.messages.length - 1 === index
             const showFeedback = this.showFeedbackFor === message.message.id && !Boolean(givenFeedback?.reason)
+
+            const htmlMessage = md.renderInline(message.message.value)
```

Comment:
will there be a performance issue in doing this at every render? if yes, then maybe we can have a local variable which is updated with the formatted values on mount and a watch() to update it at every change. 
ignore this if no performance issue.